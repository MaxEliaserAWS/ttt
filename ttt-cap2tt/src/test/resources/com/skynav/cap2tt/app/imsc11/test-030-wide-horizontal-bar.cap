Lambda字幕V4	DF0+1	SCENE"和文標準"

1	00000100/00000200	＠幅広［―］＠
2	00000200/00000300	＠倍角［―］＠
3	00000300/00000400	＠半角［―］＠
4	00000400/00000500	＠拗音［―］＠
5	00000500/00000600	＠幅［―］＠
6	00000600/00000700	＠幅０［―］＠
7	00000700/00000800	＠幅５［―］＠
8	00000800/00000900	＠幅１０［―］＠
9	00000900/00001000	＠幅１５［―］＠
10	00001000/00001100	＠幅２０［―］＠
