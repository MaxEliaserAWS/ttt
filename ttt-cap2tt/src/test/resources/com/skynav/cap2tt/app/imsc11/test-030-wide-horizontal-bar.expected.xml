<?xml version="1.0" encoding="utf-8"?><tt xmlns="http://www.w3.org/ns/ttml" ttp:contentProfiles="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttp:frameRate="30" ttp:profile="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttva:model="imsc11" xml:lang="ja" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttva="http://skynav.com/ns/ttv/annotations"><head><styling><initial tts:fontSize="4rh"/><initial tts:lineHeight="5rh"/><initial tts:showBackground="whenActive"/><style xml:id="s1" tts:textAlign="center"/><style xml:id="s2" tts:textAlign="start"/><style xml:id="s3" tts:fontSize="1.0em"/></styling><layout><region xml:id="横下" tts:displayAlign="after" tts:extent="80rw 15rh" tts:position="center bottom 10rh"/></layout></head><body region="横下" xml:space="preserve"><div><p begin="00:00:01:00" end="00:00:02:00" style="s1"><span style="s2"><span style="s3">⸺</span></span></p><p begin="00:00:02:00" end="00:00:03:00" style="s1"><span style="s2"><span style="s3">―</span></span></p><p begin="00:00:03:00" end="00:00:04:00" style="s1"><span style="s2"><span style="s3">―</span></span></p><p begin="00:00:04:00" end="00:00:05:00" style="s1"><span style="s2"><span style="s3">―</span></span></p><p begin="00:00:05:00" end="00:00:06:00" style="s1"><span style="s2"><span style="s3">―</span></span></p><p begin="00:00:06:00" end="00:00:07:00" style="s1"><span style="s2"><span style="s3">―</span></span></p><p begin="00:00:07:00" end="00:00:08:00" style="s1"><span style="s2"><span style="s3">―</span></span></p><p begin="00:00:08:00" end="00:00:09:00" style="s1"><span style="s2"><span style="s3">―</span></span></p><p begin="00:00:09:00" end="00:00:10:00" style="s1"><span style="s2"><span style="s3">⸺</span></span></p><p begin="00:00:10:00" end="00:00:11:00" style="s1"><span style="s2"><span style="s3">―</span></span></p></div></body></tt>